src/GeolocatorService/LocationPermissionChangedEventArgs.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GeolocatorService
+{
+	public class LocationPermissionChangedEventArgs : EventArgs
+	{
+		public LocationPermissionChangedEventArgs(bool isLocationPermissionAllowed)
+		{
+			IsLocationPermissionAllowed = isLocationPermissionAllowed;
```

Comment:
We use Granted in the interface (instead of Allowed), I think we should use the same term everywhere :)

Reply:
+1; could also just use simply "Granted" without the "IsSomethingSomething".